content/courses/advanced-ada/chapters/strings.rst

```diff
@@ -1688,11 +1688,134 @@ then the actual information for the derived type is displayed.
 Universal text buffer
 ---------------------
 
-.. admonition:: Relevant topics
+In the :ref:`previous section <Adv_Ada_Put_Image_Aspect>`, we've seen that the
+first parameter of the procedure indicated in the :ada:`Put_Image` aspect has
+the :ada:`Root_Buffer_Type'Class` type, which is defined in the
+:ada:`Ada.Strings.Text_Buffers` package. In this section, we talk more about
+this type and additional procedures associated with this type.
+
+.. note::
+
+   This feature was introduced in Ada 2022.
+
+Overview
+~~~~~~~~
+
+We use the :ada:`Root_Buffer_Type'Class` type to implement an universal text
```

Comment:
"an universal" should be "a universal"

Reply:
It's probably not the first time I do this mistake. ;-) Thanks for catching it!